src/services/goals.js

```diff
@@ -203,19 +203,22 @@ export async function copyGoalsToGrants(goals, grantIds, transaction) {
 
 export async function updateGoalStatusById(
   goalId,
+  oldStatus,
   newStatus,
   closeSuspendReason,
   closeSuspendContext,
 ) {
   /* TODO:
-    Disable for now until goals are unique grants.
+    Disable for now until goals are unique grants. ?????
   */
```

Comment:
Don't exactly understand this (as far as what we are waiting for), so I've left it out for now, but will gladly finish the job here if/when this is resolved

Reply:
It was disabled to prevent us (or other users) accidentally setting the goal status of a lot of different goals at once. Since a single goal can be "owned" by any number of recipients setting the goal status for a single recipient actually sets it for all recipients with that goal.